data/types/base/selfmonitored/self_monitored.go

```diff
@@ -9,8 +9,9 @@ import (
 type BloodGlucose struct {
 	base.Base `bson:",inline"`
 
-	Value *float64 `json:"value,omitempty" bson:"value,omitempty"`
-	Units *string  `json:"units,omitempty" bson:"units,omitempty"`
+	Value   *float64 `json:"value,omitempty" bson:"value,omitempty"`
+	Units   *string  `json:"units,omitempty" bson:"units,omitempty"`
+	SubType *string  `json:"subType,omitempty" bson:"subType,omitempty"`
```

Comment:
yeah I agree - there are a few inconsistencies with the data-model. Maybe we could rename the struct field and then migrate the data-model type over time?